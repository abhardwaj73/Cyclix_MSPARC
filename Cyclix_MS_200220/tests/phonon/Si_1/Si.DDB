***************************************************************************
                            Phononic properties                            
***************************************************************************

Dynamical matrix before symmetrization (not shifted) (Ha/Bohr^2) for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.3306398948 +       0.0000000000i	      0.0000380636 +      -0.0000000000i	      0.0193117236 +      -0.0000000000i	      0.3092633310 +       0.0000000000i	      0.1280563413 +       0.0000000000i	     -0.0193260041 +       0.0000000000i	
      0.0000380636 +       0.0000000000i	     -1.5149897099 +       0.0000000000i	      0.0013703906 +       0.0000000000i	     -0.5787096329 +      -0.0000000000i	      1.3971610989 +       0.0000000000i	     -0.0013691900 +      -0.0000000000i	
      0.0193117236 +      -0.0000000000i	      0.0013703906 +       0.0000000000i	     -0.0457646671 +       0.0000000000i	      0.0184055720 +      -0.0000000000i	      0.0061257190 +       0.0000000000i	      0.0456916811 +      -0.0000000000i	
      0.3092633310 +       0.0000000000i	     -0.5787096329 +      -0.0000000000i	      0.0184055720 +       0.0000000000i	     -0.5039825757 +       0.0000000000i	      0.4186743705 +      -0.0000000000i	     -0.0184165868 +       0.0000000000i	
      0.1280563413 +       0.0000000000i	      1.3971610989 +      -0.0000000000i	      0.0061257190 +      -0.0000000000i	      0.4186743705 +       0.0000000000i	     -1.3414699084 +      -0.0000000000i	     -0.0061288201 +      -0.0000000000i	
     -0.0193260041 +       0.0000000000i	     -0.0013691900 +      -0.0000000000i	      0.0456916811 +      -0.0000000000i	     -0.0184165868 +       0.0000000000i	     -0.0061288201 +      -0.0000000000i	     -0.0457393319 +      -0.0000000000i	

Dynamical matrix after symmetrization (not shifted) (Ha/Bohr^2) for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.3306398948 +       0.0000000000i	      0.0000380636 +      -0.0000000000i	      0.0193117236 +       0.0000000000i	      0.3092633310 +       0.0000000000i	      0.1280563413 +      -0.0000000000i	     -0.0193260041 +      -0.0000000000i	
      0.0000380636 +       0.0000000000i	     -1.5149897099 +       0.0000000000i	      0.0013703906 +      -0.0000000000i	     -0.5787096329 +      -0.0000000000i	      1.3971610989 +       0.0000000000i	     -0.0013691900 +       0.0000000000i	
      0.0193117236 +      -0.0000000000i	      0.0013703906 +       0.0000000000i	     -0.0457646671 +       0.0000000000i	      0.0184055720 +      -0.0000000000i	      0.0061257190 +       0.0000000000i	      0.0456916811 +      -0.0000000000i	
      0.3092633310 +      -0.0000000000i	     -0.5787096329 +       0.0000000000i	      0.0184055720 +       0.0000000000i	     -0.5039825757 +       0.0000000000i	      0.4186743705 +      -0.0000000000i	     -0.0184165868 +      -0.0000000000i	
      0.1280563413 +       0.0000000000i	      1.3971610989 +      -0.0000000000i	      0.0061257190 +      -0.0000000000i	      0.4186743705 +       0.0000000000i	     -1.3414699084 +       0.0000000000i	     -0.0061288201 +       0.0000000000i	
     -0.0193260041 +       0.0000000000i	     -0.0013691900 +      -0.0000000000i	      0.0456916811 +       0.0000000000i	     -0.0184165868 +       0.0000000000i	     -0.0061288201 +      -0.0000000000i	     -0.0457393319 +       0.0000000000i	

Phonon_frequencies (cm-1) for q = [      0.0000000000       0.0000000000       0.0000000000]:
      0.0000000000 +    1687.6713764875i	      0.0000000000 +     791.2036621740i	      0.0000000000 +     314.5091105848i	      0.0000000000 +      50.0166429813i	      0.0000000000 +       7.5342721463i	    129.4357987634 +       0.0000000000i	
 Phonon modes for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.0000106586	      0.7071347903	      0.2359957165	      0.0008096687	      0.0006247154	      0.6665323430	
      0.7071204349	     -0.0000150373	     -0.0006745243	      0.7070924741	      0.0003414349	     -0.0005931728	
     -0.0006633399	      0.0000434649	     -0.6666247931	     -0.0001166685	      0.7072733777	      0.2353195678	
      0.2705877272	     -0.6532477506	      0.2177574325	     -0.2698878239	      0.0004495248	      0.6162724392	
     -0.6532703057	     -0.2706062841	      0.0908159328	      0.6535899360	      0.0005554577	      0.2541308720	
      0.0006633692	     -0.0000403614	      0.6665216568	     -0.0005674257	      0.7069394258	     -0.2366110004	
