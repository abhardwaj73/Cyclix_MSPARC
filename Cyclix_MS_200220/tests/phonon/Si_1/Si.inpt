#CELL: 18.870000000000001 0.392699081698724 6.45  #18.000000000000000 0.698131700797732 20  
#TWIST_ANGLE: 0.030441789278971
#FD_GRID: 63 25 25
CELL: 20.000000000000000 0.392699081698724 4.0005
TWIST_ANGLE: 0.049081250056084 #0.032724923474894
FD_GRID: 80 19 18 #67 16 23
KPOINT_GRID: 1 2 12
KPOINT_SHIFT: 0 0 0.5
BC: D C H
FD_ORDER: 12
EXCHANGE_CORRELATION: LDA_PZ
CHEB_DEGREE: 100
TOL_SCF: 1e-5
SMEARING: 0.001
ELEC_TEMP_TYPE: fd

QPOINT_GRID: 0 0 0




#CELL: 20.000000000000000 0.392699081698724 8.001
#FD_GRID: 111 27 45
#KPOINT_GRID: 1 16 1
#KPOINT_SHIFT: 0 0 0
#BC: D C P
#FD_ORDER: 12
#EXCHANGE_CORRELATION: LDA_PZ
#CHEB_DEGREE: 100
#TOL_SCF: 1e-6
#SMEARING: 0.001
#ELEC_TEMP_TYPE: fd
#QPOINT_GRID: 0 0 0