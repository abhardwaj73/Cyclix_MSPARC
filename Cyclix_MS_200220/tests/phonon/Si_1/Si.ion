#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name> <valence charge>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...


# Reminder: when changing number of atoms, change the RELAX flags accordingly
#           as well.
ATOM_TYPE: Si               # atom type followed with valence charge
N_TYPE_ATOM: 2               # number of atoms of this type
PSEUDO_POT: Si_local.psp8
COORD_FRAC:
0.588157665635968                   0                   0
0.588157665635968   0.666666666666666   0.666666666666666

#COORD_FRAC:                     # coordinates follows
#1.005072284181751                   0                   0
#1.051177212639621   0.666666666666667   0.666666666666667

#COORD_FRAC:
#0.588157665635968                   0                   0
#0.588157665635968   0.000000000000000   0.333333333333333
#0.588157665635968   0.500000000000000   0.500000000000000
#0.588157665635968   0.500000000000000   0.833333333333333

#COORD_FRAC:
#0.588157665635968                   0                   0
#0.588157665635968   0.000000000000000   0.333333333333333
#0.588157665635968   0.500000000000000   0.500000000000000
#0.588157665635968   0.500000000000000   0.833333333333333

#COORD_FRAC:
#0.592678562578426                  0.0z                   0.0

#COORD_FRAC:
#0.588157665635968                   0                   0
#0.588157665635968   0.000000000000000   0.333333333333333
#0.588157665635968   0.500000000000000   0.500000000000000
#0.588157665635968   0.500000000000000   0.833333333333333


#COORD_FRAC:
#0.542714381430827                0.0                   0.0

#COORD_FRAC:
#1.005072284181751                   0                   0
#1.051177212639621  -0.000000000000000   0.333333333333333
#1.005072284181751   0.500000000000000   0.500000000000000
#1.051177212639621   0.500000000000000   0.833333333333333

#COORD_FRAC:
#0.592678562578426                  0.0                   0.0



#COORD_FRAC:                     # coordinates follows
#1.005072284181751                   0                   0
#1.051177212639621   0.666666666666667   0.666666666666667


