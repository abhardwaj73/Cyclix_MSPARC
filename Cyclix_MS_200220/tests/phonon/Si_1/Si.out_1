***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 21-Mar-2022  Start time: 09:42:37                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 4.000500 
TWIST_ANGLE: 0.049081 
FD_GRID: 67 16 15
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 2 12
KPOINT_SHIFT: 0 0 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/Si_1/Si.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.298507 (Bohr)
Mesh spacing in y-direction        :  0.288711 (Bohr)
Mesh spacing in z direction        :  0.266700 (Bohr)
Number of symmetry adapted k-points:  12
Output printed to                  :  ../tests/phonon/Si_1/Si.out_1
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.52 0.74 6.67
Number of atoms of type 1          :  2
Estimated total memory usage       :  165.10 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
