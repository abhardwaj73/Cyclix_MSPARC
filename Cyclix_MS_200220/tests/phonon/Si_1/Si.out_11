***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 23-Mar-2022  Start time: 15:54:02                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 4.000500 
TWIST_ANGLE: 0.049081 
FD_GRID: 67 16 15
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 12
KPOINT_SHIFT: 0 0 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/Si_1/Si.out_11
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.298507 (Bohr)
Mesh spacing in y-direction        :  0.288711 (Bohr)
Mesh spacing in z direction        :  0.266700 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/phonon/Si_1/Si.out_11
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.52 0.74 6.67
Number of atoms of type 1          :  2
Estimated total memory usage       :  105.34 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9964818781E+00        4.271E-01        19.365
2            -3.0551745029E+00        1.411E-01        5.824
3            -3.0703030545E+00        6.434E-02        5.453
4            -3.0719848521E+00        4.032E-02        5.634
5            -3.0715333846E+00        5.272E-02        5.252
6            -3.0662392792E+00        7.868E-02        5.622
7            -3.0722068896E+00        7.986E-03        5.543
8            -3.0721543390E+00        6.705E-03        5.086
9            -3.0720590817E+00        1.176E-02        5.172
10           -3.0722217955E+00        4.248E-03        5.507
11           -3.0722191536E+00        4.086E-03        4.885
12           -3.0722155066E+00        4.560E-03        4.541
13           -3.0722250600E+00        3.246E-03        4.840
14           -3.0722227592E+00        2.526E-03        4.767
15           -3.0722243621E+00        1.215E-03        4.886
16           -3.0722271014E+00        3.402E-04        5.149
17           -3.0722272385E+00        2.754E-04        4.685
18           -3.0722272298E+00        1.203E-05        4.676
19           -3.0722271355E+00        5.439E-06        4.261
Total number of SCF: 19    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.0722271355E+00 (Ha/atom)
Total free energy                  : -6.1444542710E+00 (Ha)
Band structure energy              : -3.9615318506E+00 (Ha)
Exchange correlation energy        : -3.0410214344E+00 (Ha)
Self and correction energy         : -2.5541705880E+01 (Ha)
Entropy*kb*T                       : -2.0748062939E-04 (Ha)
Fermi level                        : -1.6956825930E-01 (Ha)
Average force                      :  1.8645994195E-01 (Ha/Bohr)
Maximum force                      :  1.8646435253E-01 (Ha/Bohr)
Time for force calculation         :  1.626 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  128.308 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
