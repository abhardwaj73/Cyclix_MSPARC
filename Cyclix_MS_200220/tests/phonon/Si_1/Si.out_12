***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 03-Apr-2022  Start time: 22:59:57                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 4.000500 
TWIST_ANGLE: 0.049081 
FD_GRID: 80 19 18
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 12
KPOINT_SHIFT: 0 0 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 90
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/Si_1/Si.out_12
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.250000 (Bohr)
Mesh spacing in y-direction        :  0.243125 (Bohr)
Mesh spacing in z direction        :  0.222250 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/phonon/Si_1/Si.out_12
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.86 0.76 6.89
Number of atoms of type 1          :  2
Estimated total memory usage       :  178.80 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9902602847E+00        4.156E-01        44.860
2            -3.0543731746E+00        1.435E-01        15.395
3            -3.0704370666E+00        6.055E-02        14.607
4            -3.0718987011E+00        4.018E-02        12.734
5            -3.0712587775E+00        5.436E-02        11.378
6            -3.0630192923E+00        1.059E-01        14.270
7            -3.0714319552E+00        2.763E-02        12.186
8            -3.0719620356E+00        1.572E-02        12.540
9            -3.0722332357E+00        3.219E-03        12.625
10           -3.0722465931E+00        7.702E-04        11.954
11           -3.0722435557E+00        1.255E-03        9.966
12           -3.0722444252E+00        3.973E-04        11.069
13           -3.0722440900E+00        2.096E-04        11.672
14           -3.0722442703E+00        3.739E-05        10.821
15           -3.0722443369E+00        2.034E-05        10.449
16           -3.0722442882E+00        2.080E-05        9.349
17           -3.0722442941E+00        3.172E-06        10.860
18           -3.0722442392E+00        9.264E-07        8.208
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.0722442392E+00 (Ha/atom)
Total free energy                  : -6.1444884785E+00 (Ha)
Band structure energy              : -3.9619476061E+00 (Ha)
Exchange correlation energy        : -3.0410281812E+00 (Ha)
Self and correction energy         : -2.5541822775E+01 (Ha)
Entropy*kb*T                       : -2.0768969105E-04 (Ha)
Fermi level                        : -1.6961895485E-01 (Ha)
Average force                      :  1.8645069750E-01 (Ha/Bohr)
Maximum force                      :  1.8645109542E-01 (Ha/Bohr)
Time for force calculation         :  3.934 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1451.183 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
