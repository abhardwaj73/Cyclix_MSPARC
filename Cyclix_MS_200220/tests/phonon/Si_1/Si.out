***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 21-Mar-2022  Start time: 08:57:13                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 4.000500 
TWIST_ANGLE: 0.049081 
FD_GRID: 67 16 15
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 1 12
KPOINT_SHIFT: 0 0 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 10
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/Si_1/Si.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.298507 (Bohr)
Mesh spacing in y-direction        :  0.288711 (Bohr)
Mesh spacing in z direction        :  0.266700 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/phonon/Si_1/Si.out
Total number of atom types         :  1
Total number of atoms              :  2
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.52 0.74 6.67
Number of atoms of type 1          :  2
Estimated total memory usage       :  105.34 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9964818781E+00        4.271E-01        19.958
2            -3.0551745029E+00        1.411E-01        6.198
3            -3.0703030545E+00        6.434E-02        5.783
4            -3.0719848522E+00        4.032E-02        5.887
5            -3.0715333846E+00        5.272E-02        5.619
6            -3.0662392793E+00        7.868E-02        5.728
7            -3.0722068896E+00        7.986E-03        5.930
8            -3.0721543390E+00        6.705E-03        5.647
9            -3.0720590818E+00        1.176E-02        5.439
10           -3.0722217955E+00        4.248E-03        5.991
11           -3.0722191536E+00        4.086E-03        5.174
12           -3.0722155069E+00        4.560E-03        4.630
13           -3.0722250598E+00        3.246E-03        5.227
14           -3.0722227595E+00        2.526E-03        5.143
15           -3.0722243621E+00        1.216E-03        5.169
16           -3.0722271008E+00        3.402E-04        5.035
17           -3.0722272384E+00        2.755E-04        4.822
18           -3.0722272299E+00        1.202E-05        4.752
19           -3.0722271339E+00        5.442E-06        4.288
Total number of SCF: 19    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.0722271339E+00 (Ha/atom)
Total free energy                  : -6.1444542677E+00 (Ha)
Band structure energy              : -3.9615318482E+00 (Ha)
Exchange correlation energy        : -3.0410214347E+00 (Ha)
Self and correction energy         : -2.5541705880E+01 (Ha)
Entropy*kb*T                       : -2.0748061689E-04 (Ha)
Fermi level                        : -1.6956825907E-01 (Ha)
Average force                      :  1.8645994844E-01 (Ha/Bohr)
Maximum force                      :  1.8646435912E-01 (Ha/Bohr)
Time for force calculation         :  1.839 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  141.531 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
